### ISSUE TITLE
A couple more questions...

### ISSUE THREAD

**Issue by rmosolgo** [Stanko/skyblue#11]
Hi again, 

I've been working on making skyblue Rails-friendly, I hope that's ok :) https://github.com/rmosolgo/skyblue_rails

I just had a couple questions:
- The `.png` files in `img`, what are they for? Do they generate the data URIs that are used in the `select` mixin? Are they used in another way?
- I noticed there's a couple of JS files that look like compatibility fixes. What are they for?
- What do you think about using SASS `!default` for `_variables`? This would allow people to override them in external SASS files (and make it possible to override them in `skyblue-rails`)? I see this is how `boostrap-sass` does it: https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss#L11. I could PR if you're interested.

Thanks!

**Comment by rmosolgo** [comment]
lol, now I see an existing project exists: 

https://github.com/zmpeg/skyblue-rails

It doesn't support SASS though, which I'd like to do, if you're game for `!default`.

**Comment by Stanko** [comment]
Hi,
Thing is that I didn't know it will actually become popular, so there are some leftover files which should be removed or documented.
- Images are just as backup, not really doing anything. Those are used in `select` mixin, but not generated on the fly.
- Regarding JS files, as it was starting point for my own small projects, I included those two files which are basically polyfills for Internet Explorer (for HTML5 and media queries). 
- Adding `!default` makes total sense, I'll try to add this today.

New documentation probably coming next weekend.

P.S. I closed other issue as it was just an empty comment.
